src/components/Users/ManageUsers.tsx

```diff
@@ -103,7 +103,7 @@ export default function ManageUsers() {
   );
 
   if (userListLoading || districtDataLoading || !userListData?.results) {
-    return <Loading />;
+    manageUsers = <Loading />;
```

Comment:
not sure if this makes sense. 

why assign it when line 109 is anyways gonna assign it with another value anyways? this code is no-op then right?